doc/user-guides/full-walkthrough/secure-protect-connect-openshift.md

```diff
@@ -107,418 +140,261 @@ spec:
-        observedGeneration: 2
-        reason: Accepted
-        status: "True"
-        type: kuadrant.io/AuthPolicyAffected
-- lastTransitionTime: "2024-04-26T14:07:28Z"
-        message: Object affected by RateLimitPolicy demo/external
-        observedGeneration: 1
-        reason: Accepted
-        status: "True"
-        type: kuadrant.io/RateLimitPolicyAffected        
-```
-
-### Step 4 - Test connectivity and deny-all auth 
-
-You can use `curl` to hit an endpoint in the toystore app. Because you are using Let's Encrypt staging in this example, you can pass the `-k` flag as follows:
-
-```bash
-curl -s -k -o /dev/null -w "%{http_code}" "https://$(kubectl get httproute toystore -n ${devNS} -o=jsonpath='{.spec.hostnames[0]}')/v1/toys"
+kubectl apply -f - <<EOF
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: istio-proxies-monitor
+  namespace: ${KUADRANT_GATEWAY_NS}
+spec:
+  selector:
+    matchExpressions:
+      - key: istio-prometheus-ignore
+        operator: DoesNotExist
+  podMetricsEndpoints:
+    - path: /stats/prometheus
+      interval: 30s
+      relabelings:
+        - action: keep
+          sourceLabels: ["__meta_kubernetes_pod_container_name"]
+          regex: "istio-proxy"
+        - action: keep
+          sourceLabels:
+            ["__meta_kubernetes_pod_annotationpresent_prometheus_io_scrape"]
+        - action: replace
+          regex: (\d+);(([A-Fa-f0-9]{1,4}::?){1,7}[A-Fa-f0-9]{1,4})
+          replacement: "[\$2]:\$1"
+          sourceLabels:
+            [
+              "__meta_kubernetes_pod_annotation_prometheus_io_port",
+              "__meta_kubernetes_pod_ip",
+            ]
+          targetLabel: "__address__"
+        - action: replace
+          regex: (\d+);((([0-9]+?)(\.|$)){4})
+          replacement: "\$2:\$1"
+          sourceLabels:
+            [
+              "__meta_kubernetes_pod_annotation_prometheus_io_port",
+              "__meta_kubernetes_pod_ip",
+            ]
+          targetLabel: "__address__"
+        - action: labeldrop
+          regex: "__meta_kubernetes_pod_label_(.+)"
+        - sourceLabels: ["__meta_kubernetes_namespace"]
+          action: replace
+          targetLabel: namespace
+        - sourceLabels: ["__meta_kubernetes_pod_name"]
+          action: replace
+          targetLabel: pod_name
+EOF
 ```
 
-You are getting a `403` because of the existing default, deny-all `AuthPolicy` applied at the Gateway. You can override this for your `HTTPRoute`.
-
-Choose one of the following options:
 
-- API key auth flow
-- OpenID Connect auth flow 
+### Override the Gateway's deny-all AuthPolicy 
 
-### Step 5 - Set up API key auth flow
+### Set up API key auth flow
 
-Set up an example API key in each cluster as follows:
+Set up an example API key for the new users:
 
 ```bash
 kubectl apply -f - <<EOF
 apiVersion: v1
 kind: Secret
 metadata:
   name: toystore-api-key
-  namespace: ${devNS}
+  namespace: ${KUADRANT_DEVELOPER_NS}
```

Comment:
seem to be missing creating two secrets for the two users here?

e.g. https://github.com/Kuadrant/kuadrant-operator/blob/9646edb410519330a1c89559b505c9da226f0801/doc/user-guides/full-walkthrough/secure-protect-connect-k8s.md#override-the-gateways-deny-all-authpolicy